scene/gui/rich_text_label.cpp

```diff
@@ -1136,9 +1136,9 @@ void RichTextLabel::_gui_input(Ref<InputEvent> p_event) {
 				emit_signal("meta_hover_started", meta);
 			}
 		} else if (meta_hovering) {
-			emit_signal("meta_hover_ended", current_meta);
 			meta_hovering = NULL;
 			current_meta = false;
+			emit_signal("meta_hover_ended", current_meta);
```

Comment:
This makes the signal always get `false` as argument, and not the previous `current_meta` as used previously. Not sure if that's an issue.

Reply:
Oh! My bad. Fixed that, but I accidentally closed the PR :smile:. Opened a new one #27596